workloads/spec-reference.md

```diff
@@ -0,0 +1,1095 @@
+resources in the ClusterDelivery.
+
+Parameters:
+
+<table>
+  <tr>
+    <th>parameter name</th>
+    <th>meaning</th>
+    <th>example</th>
+  </tr>
+
+  <tr>
+    <td><code>gitImplementation</code></td>
+    <td>
+      VMware recommends that you use the underlying library for fetching the
+      source code.  Either <code>libggit2</code>, required for Azure DevOps, or
+      <code>go-git</code>.
+    </td>
+    <td>
+      <pre>
+      - name: gitImplementation
+        value: libggit2
+      </pre>
+    </td>
+  </tr>
+
+  <tr>
+    <td><code>gitops_ssh_secret</code></td>
+    <td>
+      Name of the secret in the same namespace as the `Deliverable` used for
+      providing credentials for fetching Kubernetes configuration files from
+      the Git repository pointed at.
+
+      See [Git authentication](git-auth).
+    </td>
+    <td>
+      <pre>
+      - name: gitops_ssh_secret
+        value: git-credentials
+      </pre>
+    </td>
+
+  </tr>
+</table>
+
+> **Note** It might not be necessary to change the default Git implementation
+> but some providers, such as Azure DevOps, require you to use `libgit2` due to
+> the server-side implementation providing support only for [git's v2
+> protocol](https://git-scm.com/docs/protocol-v2). For information about the
+> features supported by each implementation, see [git
+> implementation](https://fluxcd.io/flux/components/source/gitrepositories/#git-implementation) in the flux
+> documentation.
+
+For information about how to create a Workload that uses a GitHub
+repository as the provider of source code, see [Create a workload from GitHub
+repository](../cli-plugins/apps/create-workload.hbs.md#-create-a-workload-from-github-repository).
+
+For information about GitRepository objects, see
+[GitRepository](https://fluxcd.io/flux/components/source/gitrepositories/).
+
+### ImageRepository
+
+An ImageRepository object is instantiated when
+`deliverable.spec.source.image` is configured to continuously
+look for Kubernetes configuration files pushed to a container image registry
+as opposed to a Git repository.
+
+Parameters:
+
+<table>
+  <tr>
+    <th>parameter name</th>
+    <th>meaning</th>
+    <th>example</th>
+  </tr>
+
+  <tr>
+    <td><code>serviceAccount</code></td>
+    <td>
+      Name of the service account, in the same namespace as the Deliverable, you
+      want to use to provide the necessary permissions for `kapp-controller` to
+      deploy the objects to the cluster.
+    </td>
+    <td>
+      <pre>
+      - name: serviceAccount
+        value: default
+      </pre>
+    </td>
+  </tr>
+
+</table>
+
+For information about custom resource details, see [ImageRepository reference
+docs](../source-controller/reference.hbs.md#imagerepository).
+
+> **Note** `--service-account` flag sets the `spec.serviceAccountName` key in
+> the Deliverable object. To configure the `serviceAccount` parameter, use
+> `--param serviceAccount=...`.
+
+## app deployer
```

Comment:
Should this be listed in the 'List of supply chain resources' table above?

Reply:
Yes. Output type for this is `N/A`, `Yes` for the Basic/Testing/Scanning columns. Description should be ```Applies configuration produced by a supply chain to the cluster```

Reply:
OK, so if I hope I understand correctly, -
- I've moved app deployer up to be a H3 sub heading under  H2 Workload Parameter Reference
- I've added it to the end of the List of supply chain resources table with the inputs you provided above
Is this correct? Please confirm.